tests/test_wave_07.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+from swap_meet.clothing import Clothing
+from swap_meet.electronics import Electronics
+from swap_meet.decor import Decor
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+
+def test_get_min_age():
+    item_a = Decor(condition=3.5)
+    item_b = Electronics(condition=3.5)
+    item_c = Clothing(condition=3.5, age=9.0)
+    item_d = Item(condition=3.5, age=1.0)
+    item_e = Clothing(condition=3.5, age=6.0)
+
+    kate = Vendor(
+        inventory=[item_a, item_b, item_c, item_d, item_e]
+    )
+
+    result = kate.get_min_age()
+
+    assert result.category == "Electronics"
+    assert result.age == pytest.approx(0.0)
+
+def test_get_min_age_return_none():
```

Comment:
Good check of an edge-case.  I also suggest testing what happens if two products in inventory have the same age.